pkgs/applications/virtualization/kvmtool/default.nix

```diff
@@ -1,13 +1,13 @@
-{ stdenv, fetchzip, lib, dtc }:
+{ stdenv, fetchgit, lib, dtc }:
 
 stdenv.mkDerivation {
   pname = "kvmtool";
   version = "unstable-2024-06-14";
 
-  src = fetchGit {
-    url = "https://github.com/kvmtool/kvmtool.git";
-    ref = "master";
+  src = fetchgit {
```

Comment:
fetchGit

Reply:
Sorry for the mess.

Reply:
PR still shows this line with `fetchgit`. This is *not resolved.*

If you edited your commit history (`git rebase`, `lazygit`), you must `git push -f`

Reply:
Thanks for instructions!

Reply:
It's still `fetchgit`. Please keep trying!